pipelines/market_intelligence/cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+steps:
+  # - name: 'gcr.io/cloud-builders/gsutil'
+  #   script: |
+  #     echo Copying Gemma model from $_GCS_GEMMA_PATH
+  #     gsutil -m -q cp -r $_GCS_GEMMA_PATH /workspace
+  #     echo All files copied.
+  - name: 'gcr.io/cloud-builders/docker'
+    script: |
+      docker build -t ${_TAG} .
+substitutions:
+    # _GCS_GEMMA_PATH: unset
```

Comment:
Remove commented out line

Reply:
Removed the line and pushed the changes.